src/site/stages/build/process-cms-exports/schemas/output/node-health_services_listing.js

```diff
@@ -22,7 +25,15 @@ module.exports = {
     fieldMetaTitle: { type: 'string' },
     fieldOffice: {
       type: 'object',
-      properties: { entity: { $ref: 'output/node-health_care_region_page' } },
+      properties: {
+        /* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
You should be good to remove this disable now.

Reply:
Whoops, forgot about this one.